cirq-web/cirq_ts/src/bloch_sphere/components/Axes.class.ts

```diff
@@ -0,0 +1,46 @@
+import {Vector3, LineBasicMaterial, BufferGeometry, Line} from 'three';
+
+export class Axes {
+  /**
+   * Creates the x, y, and z axis for the bloch_sphere.
+   * @param radius The overall radius of the bloch sphere.
+   * @returns An object mapping the name of the axis to its corresponding
+   * Line object to be rendered by the three.js scene.
+   */
+  public static createAxes(radius: number) {
+    const xAxis = [new Vector3(-radius, 0, 0), new Vector3(radius, 0, 0)];
+    const yAxis = [new Vector3(0, 0, -radius), new Vector3(0, 0, radius)];
+    const zAxis = [new Vector3(0, -radius, 0), new Vector3(0, radius, 0)];
+    const colors = {
```

Comment:
My understanding is that these axes should be a standard color across all bloch spheres. (Eg. x is blue, y is red, z is green) so that the user has an additional indicator when comparing states. In what cases do you think they should be configurable? Or by configurable do you mean able to be changed within the code easier, but to whatever color we determine?